diesel_derives/src/insertable.rs

```diff
@@ -59,7 +85,23 @@ pub fn derive(item: syn::DeriveInput) -> Result<quote::Tokens, Diagnostic> {
     ))
 }
 
-fn field_ty(field: &Field, table_name: syn::Ident) -> syn::Type {
+fn direct_field_ty(field: &Field, table_name: syn::Ident) -> syn::Type {
+    if field.has_flag("embed") {
+        let field_ty = &field.ty;
+        parse_quote!(#field_ty)
```

Comment:
This can just be `field.ty.clone()`, can't it?

Reply:
Why? This line is required because of the `parse_quote!` below. We can change it to `field.ty.clone()` but I do not see the point to do this.